subsurface-mobile-main.cpp

```diff
@@ -64,6 +65,7 @@ int main(int argc, char **argv)
 
 	// some hard coded settings
 	qPrefDisplay::set_animation_speed(0); // we render the profile to pixmap, no animations
+	qPrefCloudStorage::set_save_password_local(true);
```

Comment:
this value isn't used at all in Subsurface-mobile.
https://github.com/Subsurface-divelog/subsurface/blob/967f0e56d79c4c25e54619ef8f56770baa9edaa8/mobile-widgets/qmlmanager.cpp#L540
Therefor setting it here makes no sense, nor does the commit message that assumes that this is a thing.

Reply:
that is actually not true. if save_password_local == false it will ask you for a password every time. it is however true it is not used in QML directly, but as mentioned indirectly.